beetsplug/info.py

```diff
@@ -184,6 +183,7 @@ def run(self, lib, opts, args):
         included_keys = []
         for keys in opts.included_keys:
             included_keys.extend(keys.split(','))
+        included_keys = [k for k in included_keys if k != 'path'] # Drop path even if user provides it multiple times
```

Comment:
[This line exceeds our line length limit of 79 characters](https://travis-ci.org/beetbox/beets/jobs/460820041#L1086), I'd suggest moving the comment onto the preceding line;

```suggestion
        # Drop path even if user provides it multiple times
        included_keys = [k for k in included_keys if k != 'path']
```